src/js/utils/behavior.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const assign = require('object-assign');
+const forEach = require('array-foreach');
+const Behavior = require('receptor/behavior');
+
+const sequence = function () {
```

Comment:
😬 Perhaps explore using `jest` and its babel runner: https://github.com/facebook/jest#using-babel